package.json

```diff
@@ -30,7 +30,7 @@
     "@uniswap/v3-core": "1.0.1",
     "@uniswap/v3-periphery": "1.4.3",
     "@uniswap/v3-sdk": "3.9.0",
-    "@uniswap/widgets": "2.29.2",
+    "@uniswap/widgets": "2.24.0",
```

Comment:
even though they're point releases, the ones after this have some breaking changes. Let's sit tight on the swap widget for now.